### PR TITLE
Fix/image names in templates

### PR DESCRIPTION
**Pull request by primal100**
I want to merge this change because...

Fix some minor image reference errors in templates. One is arrow_left instead of arrow-left which doesn't exist and was causing errors in the nginx logfile. And the other is to use the login images for srcset instead of checkout.

### Pull Request Checklist

(Please keep this section. It will make maintainer's life easier.)

1. [x ] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [ ] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

File: templates/dashboard/category/form.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     </li>
     <li class="back-mobile">
       <a href="{% url 'dashboard:category-list' %}" class="breadcrumbs--ellipsed-item">
-        <svg data-src="{% static "dashboard/images/arrow-left.svg" %}" fill="#fff" width="20px" height="20px" />
```
Comment (Pacu2) [inline templates/dashboard/category/form.html]: We need to rename this file as well, don't we?

File: templates/checkout/login.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
       <div class="checkout__login__continue">
         <h3>{% trans "New customer?" context "Checkout login page title" %}</h3>
         <img class="checkout-pirate" src="{% static 'images/pirate_login.png' %}"
-             srcset="{% static 'images/pirate-checkout.png' %} 1x, {% static 'images/pirate-checkout2x.png' %} 2x">
```
Comment (Pacu2) [inline templates/checkout/login.html]: They are some other places, where this image is used, I think we should:
1. Rename this file
2. Update the name in all relevant places

Comment (elwoodxblues) [reply templates/checkout/login.html]: Well, yes, a lot of static files and templates use dashes and underscores interchangeably which should be obviously unified. We should open a new issue for that and do it in one batch in a new PR.

**Comment by elwoodxblues** [comment]
I'll merged it as it fixes references to files. As for the consistency of names we have a separate issue: #1553.

**Pull request merged**